modules/topicsFpdModule.js

```diff
@@ -0,0 +1,65 @@
+import {logError, mergeDeep} from '../src/utils.js';
+import {getRefererInfo} from '../src/refererDetection.js';
+import {submodule} from '../src/hook.js';
+
+export const TOPICS_TAXONOMY = 600;
+
+export function getTopicsData(name, topics) {
+  return Object.entries(
+    topics.reduce((byTaxVersion, topic) => {
+      const taxv = topic.taxonomyVersion;
+      if (!byTaxVersion.hasOwnProperty(taxv)) byTaxVersion[taxv] = [];
+      byTaxVersion[taxv].push(topic.topic);
+      return byTaxVersion;
+    }, {})
+  ).map(([taxv, topics]) => {
+    const datum = {
+      ext: {
+        segtax: TOPICS_TAXONOMY,
+        segclass: taxv
+      },
+      segment: topics.map((topic) => ({id: topic.toString()}))
+    };
+    if (name != null) {
+      datum.name = name;
+    }
+    return datum;
```

Comment:
testing on Chrome beta, when you get topics like this
`[{"configVersion":"chrome.1","modelVersion":"2206021246","taxonomyVersion":"1","topic":301,"version":"chrome.1:1:2206021246"},{"configVersion":"chrome.1","modelVersion":"2206021246","taxonomyVersion":"1","topic":3,"version":"chrome.1:1:2206021246"}]`
and datum looks like this
`[{"name":"domain","ext":{"segtax":600,"segclass":"1"},"segment":[{"id":"301"},{"id":"3"}]}]`

Reply:
yes, it was placeholder

Reply:
segclass should be "2206021246" here @dgirardi , not 1